vuepress/docs/next/tutorials/devops/entando-de-app.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 2
+---
+
+# Configure the Entando App Context
+This tutorial describes how to modify the [EntandoApp CR](../../docs/consume/entandoapp-cr.md) to customize the ingress path of an Entando App Engine deployment. 
+
+## Prerequisites
+- An existing deployment of an Entando App or the ability to create one.
+    - If you haven't created a deployment or don't have a YAML file for an Entando deployment, follow the [Quickstart instructions](../../docs/getting-started/README.md).
+
+## Remove or Replace entando-de-app
+
+The ingress path of the App Engine currently defaults to **/entando-de-app**, which is visible in the URL. Follow the steps below to remove or replace this value.
+
+1. Open the `entando-app.yaml`, which contains several default parameters:
```

Comment:
"which contains several default parameters" << Don't think this is necessary

Reply:
arguable... it's worth hinting that the yaml doesn't include a comprehensive list of all available parameters and that the one we'll be modifying isn't a default. also, this tutorial is so short i'm stretching what i can...